addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml

```diff
@@ -105,14 +105,19 @@
         <div class="paymentmethods-container overflow-y-auto flex-grow-1">
             <div class="paymentmethods d-flex flex-column">
                 <h4 class="title-category pt-3 pb-2 px-3 m-0 bg-view border-bottom border-2">Payment method</h4>
-                <t t-foreach="payment_methods_from_config" t-as="paymentMethod" t-key="paymentMethod.id">
-                    <div class="button paymentmethod btn btn-light rounded-0 border-bottom" t-on-click="() => this.addNewPaymentLine(paymentMethod)">
-                        <div class="payment-method-display d-flex align-items-center flex-grow-1">
+                <t t-foreach="payment_methods_from_config.toSorted((a, b) => (a.use_payment_terminal === b.use_payment_terminal) ? 0 : a.use_payment_terminal ? -1 : 1)" t-as="paymentMethod" t-key="paymentMethod.id">
+                    <div class="d-flex">
```

Comment:
instead of having this div inside the `t` tag you can replace the `t` tag with a `div`
```suggestion
     <div class="d-flex" t-foreach="payment_methods_from_config.toSorted((a, b) => a.use_payment_terminal ? -1 : 1" t-as="paymentMethod" t-key="paymentMethod.id">
```
Having fewer tags, in general, is nicer.